src/backend/c-lisp.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+import json
+import sys
+from utils.random_names import random_label
+
+
+class CodegenError(Exception):
+    pass
+
+
+symbol_types = {}
```

Comment:
what is this for?

Reply:
create a class if you have to do this.

Reply:
Tracking C-Lisp variable types. So that we can enforce C type rules.

Reply:
Move everything to a class like in llvm.py

Reply:
Changed